pkg/iris/mapping.go

```diff
@@ -88,19 +88,17 @@ func (m *mapper) MapDeployments(deployments *appsv1.DeploymentList, services *v1
 }
 
 func (m *mapper) CreateDeployment(deploymentService string, deployment appsv1.Deployment) models.Deployment {
-	var service *models.Service = nil
+	var service = ""
 	if deploymentService != "" {
-		service = &models.Service{
-			Name: deploymentService,
-		}
+		service = deploymentService
 	}
 	mappedDeployment := models.Deployment{
-		Service:   service,
-		Image:     deployment.Spec.Template.Spec.Containers[0].Image,
-		Name:      deployment.Name,
-		Labels:    deployment.ObjectMeta.Labels,
-		Timestamp: deployment.CreationTimestamp.UTC().Format(time.RFC3339),
-		Properties: models.Properties{
+		ServiceName:    service,
+		Image:          deployment.Spec.Template.Spec.Containers[0].Image,
+		DeploymentName: deployment.Name,
+		Labels:         deployment.ObjectMeta.Labels,
+		Timestamp:      deployment.CreationTimestamp.UTC().Format(time.RFC3339),
+		DeploymentProperties: models.DeploymentProperties{
```

Comment:
Do we rename it to deploymentProperties so it can like that in the json? 
Maybe we can use the "json" directive for that

Reply:
I renamed it to "DeploymentProperties" for a bit of clarity since it is inside the Deployment struct which contains other 'properties'/attributes of the deployment.
The json is already called "deploymentProperties"